packages/cli/templates/default-react-plugin-package/src/index.ts.hbs

```diff
@@ -0,0 +1,12 @@
+/***/
```

Comment:
What's this noise? I see that the other index files in the repo have the same thing, and you're kind of selectively using it or not using it in this PR

Reply:
Yeah it's a bit silly, it's because the notice lint fix removes the first comment in the file if there's a leading comment, so this will be removed if the notice rule is installed, or stay if not. Can't really put a comment here explaining the situation 😅

Reply:
Best way to fix this would be to fix the lint rule itself